ui/src/testing/stubs/workspaces-api-stub.ts

```diff
@@ -50,4 +114,48 @@ export class WorkspacesApiStub extends WorkspacesApi {
     });
   }
 
+
+  getWorkspaces(options?: any): Promise<WorkspaceResponseListResponse> {
+    return new Promise<WorkspaceResponseListResponse>(resolve => {
```

Comment:
nit: Probably more readable to just get the value then do a `return Promise.resolve(value)`.